skim.py

```diff
@@ -459,6 +469,7 @@ def apply_keep(array, sel, keep):
         parser = argparse.ArgumentParser()
         parser.add_argument('--stageout', type=str, help='stageout directory', required=True)
         parser.add_argument('-k', '--keep', type=float, default=None)
+        parser.add_argument("--skip_cut", type=str, help="Specify a selection to skip", choices=["rt", "muon_veto", "electron_veto", "metdphi", "highpt_muon_veto"])
```

Comment:
`highpt_muon_veto` is just a noise/misreco cut, so I don't think we'll make a dedicated N-1 plot for it